stories/atoms/components/icon/icon.stories.mdx

```diff
@@ -178,6 +178,37 @@ As seen in the examples below you can rotate icons, add background colors, anima
   </Story>
 </Canvas>
 
+## Invert
+
+The **invert** property allows you to invert the theme of the icon.  
```

Comment:
Properties should be highlighted using back tics as per the [wiki](https://github.com/okp4/ui/wiki/Create-A-Story).